poodledo/apiclient.py

```diff
@@ -336,8 +338,8 @@ def get_access_token(self):
                               client_secret=self.application_token,
                               code=code,
                               vers='0.2',  # TODO: code duplication.
-                              device=os.uname().sysname,
-                              os=os.uname().release)
+                              device=platform.uname().release,
```

Comment:
I think this is supposed to be `device=platform.uname().system`. Right?

Reply:
Yes. Fixed that. Thank you!